### PR TITLE
Add 2.2 release post

### PR DESCRIPTION
**Pull request by mrocklin**

### CODE REVIEW

File: _posts/2019-07-31-dask-2.2.md [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+layout: post
+title: Dask Release 2.2.0
+tags: [release]
+theme: twitter
+---
+{% include JB/setup %}
+
+I'm pleased to announce the release of Dask version 2.2.
+This is a significant release with bug fixes and new features.
+The blogged release was 2.0 on 2019-06-22.
```
Comment (TomAugspurger) [inline _posts/2019-07-31-dask-2.2.md]: typo here. Maybe missing "last"

File: _posts/2019-07-31-dask-2.2.md [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+
+```python
+import dask.dataframe as dd
+
+df = dd.read_parquet("/path/to/mydata.parquet", engine="arrow")
+# or
+df = dd.read_parquet("/path/to/mydata.parquet", engine="fastparquet")
+```
+
+Supporting for both libraries within Dask has been helpful for
```
Comment (TomAugspurger) [inline _posts/2019-07-31-dask-2.2.md]: typo: remove "for"

File: _posts/2019-07-31-dask-2.2.md [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+> Opening dataset and df.head() takes 7 - 30s. (Presumably depending on whether s3fs cache is cold or not). THIS IS HUGE! This was literally impossible before.
+
+The API remains the same, but functionality should be smoother.
+
+Thanks to [Rick Zamora](https://github.com/rjzamora), [Martin
+Durant](https://github.com/martindurant) for doing most of the work here and to
+[Sarah Bird](https://github.com/birdsarah), [Wes
+McKinney](https://github.com/wesm), and [Mike
+McCarty](https://github.com/mccarty) for providing guidance and review.
```
Comment (mmccarty) [inline _posts/2019-07-31-dask-2.2.md]: Thanks for the mention!  Corrected profile link: https://github.com/mmccarty

File: _posts/2019-07-31-dask-2.2.md [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+As always there are too many changes to list here,
+instead we'll highlight a few that readers may find interesting,
+or that break old behavior.
+In particular we discuss the following:
+
+1.  Parquet rewrite
+2.  Nicer HTML output for Clients and Logs
+3.  async/await everywhere, and cleaner setup for developers
+4.  A new SSH deployment solution, and hints of more
+5.  Move bytes I/O handling out of Dask to FSSpec
```
Comment (TomAugspurger) [inline _posts/2019-07-31-dask-2.2.md]: Can you reorder to match the order of the sections?

**Comment by TomAugspurger** [comment]
Can you add a "Hyperband implemented in Dask-ML" section? Something like

```
Hyperband in Dask-ML
--------------------

Dask-ML 2.1 was released with a new `HyperBandSearchCV` meta-estimator for
hyperparameter optimization. This can be used as an alternative to
`RandomizedSearchCV` to find similar hyper-parameters in less time by not
wasting time on hyper-parameters that are not promising.

>>> import numpy as np
>>> from dask_ml.model_selection import HyperbandSearchCV
>>> from dask_ml.datasets import make_classification
>>> from sklearn.linear_model import SGDClassifier
>>>
>>> X, y = make_classification(chunks=20)
>>> est = SGDClassifier(tol=1e-3)
>>> param_dist = {'alpha': np.logspace(-4, 0, num=1000),
>>>               'loss': ['hinge', 'log', 'modified_huber', 'squared_hinge'],
>>>               'average': [True, False]}
>>>
>>> search = HyperbandSearchCV(est, param_dist)
>>> search.fit(X, y, classes=np.unique(y))
>>> search.best_params_
{'loss': 'log', 'average': False, 'alpha': 0.0080502}

Thanks to [Scott Sievert](http://github.com/stsievert).
```

cc @stsievert if you have any corrections.

**Comment by mrocklin** [comment]
Yes, that sounds great.  We can also link to his SciPy video.

**Comment by mrocklin** [comment]
@birdsarah do you mind being quoted here?

File: _posts/2019-07-31-dask-2.2.md [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+However, there was still code within Dask that helped to unify them a bit.
+We've moved this code out to an external project,
+[fsspec](https://filesystem-spec.readthedocs.io/en/latest) which includes all
+of the centralization code that Dask used to provide, as well as a formal
+specification for what a remote data system should look like in order to be
+compatible.  This also helps to unify efforts with other projects like Arrow.
+
+Special thanks to [Martin Durant](https://github.com/martindurant) for
+shepharding Dask's I/O infrastructure over the years, and for doing the more
+immediate work of splitting out `fsspec`.
```
Comment (martindurant) [inline _posts/2019-07-31-dask-2.2.md]: Maybe link the blog post specifically about this

File: _posts/2019-07-31-dask-2.2.md [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+
+
+Async/Await everywhere, and cleaner setup for developers
+--------------------------------------------------------
+
+In Dask 2.0 we dropped Python 2 support and now support only Python 3.5.
```
Comment (martindurant) [inline _posts/2019-07-31-dask-2.2.md]: Python >= 3.5

**Comment by birdsarah** [comment]
> @birdsarah do you mind being quoted here?

No problem

**Comment by stsievert** [comment]
> an alternative to `RandomizedSearchCV` to find similar hyper-parameters in less time by not wasting time on hyper-parameters that are not promising.

That's a really succinct description @TomAugspurger. There's more detail but it's mentioned in the talk.

File: _posts/2019-07-31-dask-2.2.md [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+We've added a second SSH cluster deployment solution.  It looks like this:
+
+```python
+from distributed.deploy.ssh2 import SSHCluster  # this will move in future releases
+
+cluster = SSHCluster(
+    hosts=["localhost", "localhost", "localhost"],
+    worker_kwargs={"nthreads": 4},
+    scheduler_kwargs={},
+    connect_kwargs={"known_hosts": None}
```
Comment (stsievert) [inline _posts/2019-07-31-dask-2.2.md]: I'd use `SSHCluster` with different hosts if I used it. I'm curious on how to do that, but this example doesn't seem to reflect that.

**Comment by mrocklin** [comment]
Thanks all.  I believed I've handled that round of comments.

**Review by TomAugspurger — APPROVED** [review]

**Pull request merged**